### ISSUE TITLE
ARIA expanded

### ISSUE THREAD

**Issue by i-like-robots** [Financial-Times/o-expander#14]
I've been using the site a little with a screen reader the last few days I think how this component is described could be improved. We're currently managing the `aria-expanded` attribute on the content container rather than the toggle which isn't wrong but it doesn't always express how the elements function together in context. In Next we are primarily using the expander to hide/show content.

The spec describes the attribute like so:

> Indicates whether the element, or another grouping element it controls, is currently expanded or collapsed.
http://www.w3.org/TR/wai-aria/states_and_properties#aria-expanded

Currently when you use an expander the toggle element by default does not have any context (unless the developer has added a label) or state conveyed (`aria-pressed` lacks a value). For instance when attempting the use the collapsed main navigation on Next the following will be read both for the inactive and active state:

> List item, menu, button

By applying the `aria-expanded` attribute and relevant state to the button the following is read, which I think provides valuable context, especially when toggling:

> List item, menu, button, collapsed
> List item, menu, button, expanded

Here's a couple of suggested implementations demonstrating the `aria-expanded` attribute applied to the toggle:

1. https://www.marcozehe.de/2010/02/10/easy-aria-tip-5-aria-expanded-and-aria-controls/
2. http://heydonworks.com/practical_aria_examples/#progressive-collapsibles

**Comment by onishiweb** [comment]
Fixed by #16
